static/js/components/tiles/map_tile.tsx

```diff
@@ -339,6 +340,7 @@ function rawToChart(
   const metadataMap = rawData.placeStat.facets || {};
   const placeStat = rawData.placeStat.data[statVarSpec.statVar] || {};
 
+  const dataDates = {};
```

Comment:
the date for each value is already in the metadata (line 345), no need to add this

Reply:
Ooh thanks for the heads up! Swapped to using `metadata` instead.